src/moderatorFrontendMain/kotlin/util/Strings.kt

```diff
@@ -733,6 +733,12 @@ object Strings {
         "Bitte kontaktieren Sie das Studo Team für weitere Unterstützung oder falls Sie weitere " +
         "Konfigurationsparameter wünschen."
   )
+
+  val live_check_ins = LocalizedString(
+    "Live check-ins",
+    "Live Check-ins",
```

Comment:
```suggestion
    "Currently checked in (live)",
    "Derzeit eingechecked (live)",
```

I think if I were a philosophy professor, I wouldn't be sure what "live check-ins" means. But also not sure if this is easier to understand.

Reply:
```suggestion
    "Live - currently checked in",
    "Live - derzeit eingecheckt",
```